### PR TITLE
trying pure css tooltips

### PR DESCRIPTION
**Pull request by matakuka**
This is an attempt to considerably improve page rendering and performance, as on some pages javascript took up to >500ms after the page was loaded. About 2/3 of this time has been tooltip rendering.

This implements similar looking tooltips in pure css, being somehow faster. The effect is visible only on lage pages with lots of tooltips, so **I am unsure if we should implement it**.

### CODE REVIEW

**Comment by toirl** [comment]
Thanks for this improvement @matakuka!. As this PR introduces some changes in the UI we will need some discussion on this with other developers too. 

From my point of view this change will improve the performance on large overview pages without pagination (e.g in advanved search) where many tooltips are used. IMO this is not a typical usecase, but it is possible that someone does this. It might even be relevant for very very large forms where the help text is also rendered as Tooltip.  This comes at the price of a changed UI where we implement the tooltips in another way than in the rest of the application. CSS vs. Booststrap.

I personally tend to hold back the change as the improvement only takes effect in a rare usecase. @matakuka has also done other improvement which has much more impact on the performance than this one. But I am totally fine with merging the change into master if the majority of the other coredev want this change. @ringo-framework/ringo-core-dev what to you think?

**Comment by ThomasJunk** [comment]
If you have a look at the [tooltip-source](https://github.com/twbs/bootstrap/blob/v4-dev/js/src/tooltip.js), you see, that some things are going on in there. A blind guess: 
[here](https://github.com/twbs/bootstrap/blob/v4-dev/js/src/tooltip.js#L460) are all the eventlisteners registered. Without further knowledge, I would assume, that for each and every tooltip, an extra listener is registered. This is IMHO costly. A better approach would be to register one Listener and use _event-delegation_ to deal with the according event: Instead of registering 1000 Listeners for 1000 Tooltips, you use only one that does the trick. Additionally there is a setTimeout registered for each tooltip, which has a cost on its own.

So my assumption is: the more tooltips you have, the slower your page gets. And: Fun on mobile :laughing: 

I am fine with a CSS approach, since we:
* do not have to support older browsers and do not need any shims 
* do not "need" fancy transitions, so handling of different transition-events for different browsers is not needed
* do not need to run on mobile ( I don't know, how hover is applied under mobile, so if we were developing for mobile we should find out :grin: )

This is the "downside", if you use _off the shelf_-solutions like bootstrap:
For most of the cases does the approach work, but not for each and every cornercase.

I would avoid buying into the jQuery-Plugin world. On the other side: you have a nice kickstart upfront. 

**tl;dr**
If we are sure, that we do not want "fancy" tooltips and do not target mobile, I would vote for @matakuka s solution. Though, I have only skimmed the source and hope, it holds :grinning:

**Comment by matakuka** [comment]
With this PR I catch all tooltips in ringo, but there is still the issue of almost identical code in formbar
(affected: templates/label.mako:22, 24 and static/js/formbar.js:973). How to cope with that? Could formbar be used without ringo or is it dependant (and thus can rely on the css statements to exist?)?

**Comment by toirl** [comment]
@matakuka Formbar can be used without ringo. So you can not rely on any css statements to exist.

**Comment by matakuka** [comment]
@toirl : In the second commit I changed all tooltips I could find in ringo to the css behaviour (most of them had the "standard browser tooltips", which are different in each browser).

**Comment by matakuka** [comment]
@toirl there's something still not complete with the second commit, please don't merge yet

**Comment by matakuka** [comment]
@toirl Unfortunately, we cannot change all tooltips. Block elements (e.g. table headers `<th>`) don't offer the option to manipulate them via css or javascript, and ignore such instructions. There we'll have to live with whatever the browser throws at us (e.g. in templates/selection.mako)
